Dockerfile

```diff
@@ -29,4 +28,6 @@ RUN if [ "$use_pre_trained_model" = "true" ] ; then \
 
 EXPOSE 5000
 
-CMD python /workspace/app.py
+USER max
```

Comment:
Do we want to repeat this line here?

Reply:
Just tested it out - you are correct. I'll remove this line